api-reference/beta/resources/security_overview.md

```diff
@@ -0,0 +1,46 @@
+# Use Microsoft Graph to integrate with the Intelligent Security Graph
+
+ > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+The Microsoft Intelligent Security Graph is a unified platform for combatting cyberthreats. It powers real-time threat protection for Microsoft products and services and supports an ecosystem of integrated solutions.
+
+The security API for the Microsoft Graph makes it easy to connect with and extend the Intelligent Security Graph. Leverage the Microsoft Graph to build security solutions that consolidate and correlate security alerts from multiple sources, unlock contextual data to inform investigations, and automate security operations for greater efficiency.
+
+## Why connect to the Intelligent Security Graph?
+
+By connecting with Microsoft and partner security solutions that are part of the Intelligent Security Graph, you can more readily realize and enrich the value of these solutions. Use the security API to:
+
+### Unify and standardize alert management
+
+Correlate alerts across security solutions more easily with a common alert schema, write code once to integrate alerts from any graph-supported security solution, and keep alert status and assignments in sync across all solutions. Also, stream alerts to Security Information and Event Management (SIEM) solutions such as Splunk and IBM QRadar via [Azure Monitor](https://blogs.msdn.microsoft.com/azuresecurity/2016/08/23/azure-log-siem-configuration-steps/).

```

Comment:
By "graph-supported security solution", which graph do you mean? This "security graph", or "Microsoft Graph"?

Reply:
"Security Information and Event Management" should be lower case.